sdk/src/driftClient.ts

```diff
@@ -4314,24 +4305,63 @@ export class DriftClient {
 		orderParams: OptionalOrderParams,
 		makerInfo?: MakerInfo | MakerInfo[],
 		referrerInfo?: ReferrerInfo,
+		bracketOrdersParams = new Array<OptionalOrderParams>(),
```

Comment:
can we have this in a separate method to avoid making a breaking change? also idt bot users will use the bracket orders/cancel exisitng orders so its nice to keep it simple